<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureSkillSubsets</class>
 <widget class="QDialog" name="ConfigureSkillSubsets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Skill Subsets</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Skill Subsets</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="lwSkillSubsets">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>167</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="bDeleteSubset">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="gui_icons.qrc">
                   <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="bAddSubset">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="gui_icons.qrc">
                   <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frmName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="leName">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maxLength">
                  <number>16</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frmType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Type</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="cboxType">
                 <item>
                  <property name="text">
                   <string>Normal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Teleport</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Escape</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Switch</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureSkillSubsets</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureSkillSubsets</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
